langchain4j-chroma/pom.xml

```diff
@@ -28,7 +28,18 @@
 
         <dependency>
             <groupId>com.squareup.retrofit2</groupId>
-            <artifactId>converter-gson</artifactId>
+            <artifactId>converter-jackson</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+
+		<dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <scope>provided</scope>
```

Comment:
Please don't introduce lombok as we are trying to get rid of it

Reply:
Hi @Martin7-1 thanks a lot for your comments! I was out during the past two weeks and unable to look into your comment. I am going to follow your comments, rebase and push it back. Thanks again!

Reply:
@Martin7-1 I have applied all your suggestions, thank you! All the tests are passing locally.